### DISCUSSION TITLE
os specific jvm args ?

### DISCUSSION THREAD

**Discussion by ennerf** [hydraulic-software/conveyor#12] [Q&A] [answered]
First of all, great work! I started porting a JavaFX app today, and so far the experience has been a lot smoother than I expected. The documentation is really good for a first release.

That being said, one issue I'm currently still struggling with is OS specific includes and JVM arguments. I'd expect `app.{windows,linux,mac}.jvm.options` or `app.jvm.{windows,linux,mac}.options` to let me pick different (e.g. garbage collector) options per OS, but neither seems to work, and I haven't found anything in the documentation.

**Comment by mikehearn** [comment]
It's not in the docs because unfortunately, that isn't supported yet. It's pretty easy to add though, so we'll get right on that. Out of curiosity, could you elaborate on which JVM options need to be specified only on specific operating systems for your app?

**Comment by ennerf** [comment]
We ran into some issues with ZGC on Linux, so we'd like to run G1GC on Linux and ZGC on Windows. There are also cases where certain options are only supported on a subset of OS.

**Reply by mikehearn** [reply]
Got it, thanks.

**Comment by mikehearn** [comment, marked as answer]
Support for machine and launcher specific JVM options shipped in Conveyor 2.
